src/mediatype-icon.ts

```diff
@@ -0,0 +1,35 @@
+import { html, LitElement } from 'lit';
+import { customElement, property } from 'lit/decorators.js';
+
+import { audioIcon } from './assets/img/icons/mediatype/audio';
+import { etreeIcon } from './assets/img/icons/mediatype/etree';
+import { imagesIcon } from './assets/img/icons/mediatype/images';
+import { filmIcon } from './assets/img/icons/mediatype/film';
+import { softwareIcon } from './assets/img/icons/mediatype/software';
+import { textsIcon } from './assets/img/icons/mediatype/texts';
+import { tvIcon } from './assets/img/icons/mediatype/tv';
+import { videoIcon } from './assets/img/icons/mediatype/video';
+import { webIcon } from './assets/img/icons/mediatype/web';
+
+@customElement('mediatype-icon')
+export class MediaTypeIcon extends LitElement {
+  @property({ type: String }) mediatype = '';
+
+  private readonly mediaTypeIcons: { [key: string]: any } = {
```

Comment:
suggestion: 

`MediaTypeIcon` =>  `MediatypeIcon`
`mediaTypeIcons` => `mediatypeIcons`

I think our use of the term `mediatype` as a "unit" term. Like `software` (not `softWare`).

Also:  the lit file name convention for a class is to have a `-` between each camelCased term (don't remember where I saw this, but could look it up).
So, `mediatype-icon.ts` => `class MediatypeIcon`
Otherwise,
`media-type-icon.ts` => `class MediaTypeIcon`

Reply:
The above lit file-name-matching-a-classname spec should probably be a blocker.

Reply:
updated PR, please check